### PR TITLE
contrib: update nsqadmin.cfg.example

### PR DESCRIPTION
**Pull request by StellarisW**
update ACL and TLS usage for nsqadmin

### CODE REVIEW

File: contrib/nsqadmin.cfg.example [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+## log message prefix (default "[nsqadmin] ")
+# log_prefix = ""
+
+## HTTP header to check for authenticated admin users (default "X_Forwarded_User")
+# acl_http_header = ""
+
+## admin user (may be given multiple times; if specified, only these users will be able to perform privileged actions; acl_http_header is used to determine the authenticated user)
+# admin_users = [
+#     "admin",
+#     "asdqwc124134134e"
```
Comment (ploxiln) [inline contrib/nsqadmin.cfg.example:13]: this is an odd example username

File: contrib/nsqadmin.cfg.example [hunk 2]
```diff
@@ -1,6 +1,42 @@
 ## log verbosity level: debug, info, warn, error, or fatal
 log_level = "info"
 
+## log message prefix (default "[nsqadmin] ")
+# log_prefix = ""
+
+## HTTP header to check for authenticated admin users (default "X_Forwarded_User")
+# acl_http_header = ""
+
+## admin user (may be given multiple times; if specified, only these users will be able to perform privileged actions; acl_http_header is used to determine the authenticated user)
```
Comment (ploxiln) [inline contrib/nsqadmin.cfg.example:10]: this help text doesn't work exactly as-is for this config format ("given multiple timpes"?)

also it should be wrapped somewhere around 80 chars depending on taste ;)

File: contrib/nsqadmin.cfg.example (hunk 1 above)
Comment (StellarisW) [reply contrib/nsqadmin.cfg.example]: i consider "admin_users" as a secret string actually, I'll change it.

Comment (ploxiln) [reply contrib/nsqadmin.cfg.example]: Ah ... that's a clever way to use it! (As you probably already know, it is intended to be used behind an auth proxy, which is trusted to always set or clear this header value, like https://github.com/oauth2-proxy/oauth2-proxy)

**Pull request merged**